Source/ACE/Entity/WorldObject.cs

```diff
@@ -72,6 +85,10 @@ public ushort ForcePositionIndex
 
         public SequenceManager Sequences { get; }
 
+        public object InventoryMutex => this.inventoryMutex;
+
+        public Dictionary<ObjectGuid, WorldObject> Inventory => this.inventory;
```

Comment:
this should be a ReadOnlyDictionary.   You'll have to new it up in the constructor.  The default for "inventory" may fire after this, rendering this default incorrect.  I'm not exactly sure when the "=>" operator binds on the property, so it may work after all.  If you feel this is correct as is, I'm open to discussion as always.